JonArnfred/week1/assignment1/HelloWorld_sans_react.html

```diff
@@ -0,0 +1,108 @@
+
+<html>
+<head>
+	<title>Hello world without react</title>
+	<style type="text/css">
+	body {
+		width: 1000px;
+		margin: auto;
+	}
+	</style>
+</head>
+<body>
+
+<!-- I assumed you wanted a form, since action is a form property, not a button property
+https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button. However, I rewrote the template markup a bit-->
+  <form id="myform">
+	  <input type="text" value="warnings">
+	  <input type="submit" value="Submit"/>
+  </form>
+	<input type="text" onKeyDown="checkInputNoWarning()" value="no warnings">
+  <button onclick="printem()">All messages so far</button> 
```

Comment:
I don't understand why you have the form here. There is no reason to as you would only need one field.
Which leads me to the question on why you have the second text-field?